<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>常用逻辑门</string>
   </property>
  </widget>
  <widget class="QLabel" name="Zgate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gates/images.qrc">:/Z.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Measuregate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gates/images.qrc">:/Measure.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Ufgate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>80</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gates/images.qrc">:/uf2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Tgate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gates/images.qrc">:/T.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Sgate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gates/images.qrc">:/S.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Xgate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gates/images.qrc">:/X.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Ygate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gates/images.qrc">:/Y.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Hgate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gates/images.qrc">:/H.png</pixmap>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>20</width>
     <height>531</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>操作区</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbit1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
	background-color: rgb(85, 0, 255); 
	border-radius: 10px; 
	color: rgb(255, 255, 255); 
} </string>
   </property>
   <property name="text">
    <string>|0&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbit2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
	background-color: rgb(85, 0, 255); 
	border-radius: 10px; 
	color: rgb(255, 255, 255); 
} </string>
   </property>
   <property name="text">
    <string>|0&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbit3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
	background-color: rgb(85, 0, 255); 
	border-radius: 10px; 
	color: rgb(255, 255, 255); 
} </string>
   </property>
   <property name="text">
    <string>|0&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbit4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
	background-color: rgb(85, 0, 255); 
	border-radius: 10px; 
	color: rgb(255, 255, 255); 
} </string>
   </property>
   <property name="text">
    <string>|0&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qbit5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
	background-color: rgb(85, 0, 255); 
	border-radius: 10px; 
	color: rgb(255, 255, 255); 
} </string>
   </property>
   <property name="text">
    <string>|0&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Con_fun">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>100</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
	background-color: rgb(100, 50, 50); 
	border-radius: 6px; 
	color: rgb(255, 255, 255); 
} </string>
   </property>
   <property name="text">
    <string>生成
常函数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bala_fun">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>200</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
	background-color: rgb(100, 50, 50); 
	border-radius: 6px; 
	color: rgb(255, 255, 255); 
} </string>
   </property>
   <property name="text">
    <string>生成
平衡函数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Check_fun">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>300</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
	background-color: rgb(100, 50, 50); 
	border-radius: 6px; 
	color: rgb(255, 255, 255); 
} </string>
   </property>
   <property name="text">
    <string>查看
当前f(x)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Do">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
	background-color: rgb(230, 0, 0); 
	border-radius: 6px; 
	color: rgb(255, 255, 255); 
} </string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>功能区</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>490</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>拖动到此
删除控件</string>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>440</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>20</width>
     <height>531</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="gates/images.qrc"/>
 </resources>
 <connections/>
</ui>
